ci/Jenkinsfile

```
@@ -17,6 +17,10 @@ pipeline {
         parallelsAlwaysFailFast()
     }
 
+    environment {
+        GH = '~/bin/gh'
```

Wonder if this would be better so `gh` doesn't necessarily need to be in the home directory (e.g. if it ends up getting installed into spack-stack or a user has it in ~/.local/bin):
```suggestion
        GH = sh(script: "which gh || echo '~/bin/gh'", returnStdout: true).trim()
```